spec/frequency_classes_service_spec.rb

```diff
@@ -0,0 +1,20 @@
+require 'frequency_classes_service'
+
+describe 'FrequencyClassesService' do
+  let(:frequency_classes_service) { FrequencyClassesService.new }
+  let(:sample_word) { 'haben' }
+
+  describe '#initialize' do
+    it 'should return a FrequencyClassesService instance' do
```

Comment:
if you add any behavior to initialize, you'll want to test that, but if you only care that new returns an instance, that's what happens by default and other tests will start failing with the same exception if that changes, most likely